parts/k8s/kubernetesinstalls.sh

```diff
@@ -19,7 +19,7 @@ installEtcd() {
     if [[ "$CURRENT_VERSION" == "${ETCD_VERSION}" ]]; then
         echo "etcd version ${ETCD_VERSION} is already installed, skipping download"
     else
-        retrycmd_get_tarball 360 10 /tmp/etcd-v${ETCD_VERSION}-linux-amd64.tar.gz ${ETCD_DOWNLOAD_URL}/etcd-v${ETCD_VERSION}-linux-amd64.tar.gz || exit $ERR_ETCD_DOWNLOAD_TIMEOUT
+        retrycmd_get_tarball 120 5 /tmp/etcd-v${ETCD_VERSION}-linux-amd64.tar.gz ${ETCD_DOWNLOAD_URL}/etcd-v${ETCD_VERSION}-linux-amd64.tar.gz || exit $ERR_ETCD_DOWNLOAD_TIMEOUT
```

Comment:
You feel good about reducing the timeout to 5 seconds here?

Reply:
that number is sleep between retries https://github.com/Azure/aks-engine/blob/master/parts/k8s/kubernetesprovisionsource.sh#L85

timeout of each retry is hardcoded in retrycmd_get_tarball as 60 seconds

5 second sleep is the same number we use for other retrycmd_get_tarball calls (eg, Azure CNI). I don't think sleeping 10s between failures is necessary.

Reply:
Got it, thanks!